gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/LaunchDagProc.java

```diff
@@ -192,6 +193,15 @@ private void submitJobToExecutor(DagManagementStateStore dagManagementStateStore
     }
   }
 
+  @Override
   protected void sendNotification(Optional<Dag<JobExecutionPlan>> result, EventSubmitter eventSubmitter) {
   }
+
+  @Override
+  protected void commit(DagManagementStateStore dagManagementStateStore, Optional<Dag<JobExecutionPlan>> result) {
+    // Only call conclude method if multi-active execution enabled
+    if (this.isMultiActiveExecutionEnabled) {
+      this.launchDagTask.conclude();
+    }
+  }
```

Comment:
let's try to have the base class implement, by adding members:
```
private DagTask dagTask;
private boolean isMAExecEnabled;
```

Reply:
We could call conclude and instead have a check at the `DagTask` level which checks if an arbiter exists, if it does then call `concludeLease` and otherwise do a no-op (because we use a dummy `LeaseObtainedStatus` for non MA execution use)

Reply:
removed the `isMAExecEnabled` member and added `dagTask` to the base class. See the changes